internal/server/models/encryption.go

```diff
@@ -58,3 +58,49 @@ func (s *EncryptedAtRest) Scan(v interface{}) error {
 
 	return nil
 }
+
+// EncryptedAtRestBytes defines a field that knows how to encrypt and decrypt itself with Gorm
+// it depends on the SymmetricKey being set for this package.
+type EncryptedAtRestBytes []byte
```

Comment:
Since `[]byte` and `string` are effectively the same , I wonder if we could have only a single one of these (`EncryptedAtRestBytes` and `EncryptedAtRest`), and add a `String()` or `Bytes()` method to convert them?

Looks like the string version is already doing `[]byte(v)`, so the convert is already happening in one case.